.vscode/cspell.json

```diff
@@ -461,6 +461,12 @@
         "undoc"
       ]
     },
+    {
+      "filename": "sdk/loadtestservice/azure-developer-loadtest/azure/developer/loadtesting/**",
+      "words":[
+        "vusers"
+      ]
+    },
```

Comment:
There seems be many words which fail cspell check: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1720898&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0. it is more convenient to add your folder here: https://github.com/Azure/azure-sdk-for-python/blob/37abf4e4da05bd776226faeb923cf3699f4548d2/.vscode/cspell.json#L12